pkg/database/upsertCluster.go

```diff
@@ -193,3 +224,25 @@ func (dao *DAO) clusterPropsUpToDate(clusterUID string, resource model.Resource)
 		return false
 	}
 }
+
+func goquDelete(tableName, columnName, arg string) (string, []interface{}, error) {
+	// Create the query
+	schemaTable := goqu.S("search").Table(tableName)
+	ds := goqu.From(schemaTable)
+	sql, args, err := ds.Delete().Where(goqu.C(columnName).Eq(arg)).ToSQL()
+	return sql, args, err
+}
+
+func goquInsertUpdate(tableName string, args []interface{}) (string, []interface{}, error) {
+	// Create the query
+	// query := "INSERT INTO search.resources as r (uid, cluster, data) values($1,'',$2)
+	// ON CONFLICT (uid) DO UPDATE SET data=$2 WHERE r.uid=$1"
```

Comment:
Move this comment to document the function.
```suggestion
// Create the upsert query
// query := "INSERT INTO search.resources as r (uid, cluster, data) values($1,'',$2)
// ON CONFLICT (uid) DO UPDATE SET data=$2 WHERE r.uid=$1"
func goquInsertUpdate(tableName string, args []interface{}) (string, []interface{}, error) {
```

Reply:
Done